scipy/stats/tests/test_sampling.py

```diff
@@ -319,6 +319,30 @@ def check_discr_samples(rng, pv, mv_ex):
     assert pval > 0.1
 
 
+def test_warning_center_not_in_domain():
```

Comment:
Why is this one not parametrized like the other?

Reply:
Center is only used by `NumericalInversePolynomial'